components/select/index.en-US.md

```diff
@@ -47,6 +47,9 @@ Select component to select value from options.
 | showSearch | Whether show search input in single mode. | boolean | false |
 | size | Size of Select input. `default` `large` `small` | string | default |
 | suffixIcon | The custom suffix icon | ReactNode | - |
+| removeIcon | The custom remove icon | ReactNode | - |
+| clearIcon | The custom clear icon | ReactNode | - |
+| menuItemSelectedIcon | The custom menuItemSelected icon | ReactNode | - |
```

Comment:
Do document there props cause we don't encourage people to customize them.

Reply:
hmm, so do I need to remove them?

Reply:
yep

Reply:
@afc163  done. the commit of updating doc has been dropped now.